docs/fundamentals/gateway/custom-gateway.md

```diff
@@ -0,0 +1,326 @@
+(custom-gateway)=
+# Customize the Gateway
+A Jina Gateway is customizable and can be implemented in a similar way Executors are implemented.
```

Comment:
"...in a similar way _to how_ Executors are implemented."

Reply:
Better still would be to depassivize the entire sentence: "Jina Gateways are customizable and you can implement them in much the same way as an Executor."